integrations/confluence-ariada/recovered/static/app/styles.css

```
@@ -0,0 +1,65 @@
+:root {
+  --ink: #172b4d;
+  --muted: #5e6c84;
+  --paper: #fffdf7;
+  --line: #d8d3c5;
+  --navy: #0b3b57;
+  --blue: #0c66e4;
+  --amber: #f4b63f;
+  --pass: #16794a;
+  --danger: #ae2e24;
+  font-family: "Charlie Text", "Aptos", sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)